operator/cmd/seldon/cli/unload.go

```diff
@@ -0,0 +1,62 @@
+package cli
+
+import (
+	"k8s.io/utils/env"
+
+	"github.com/seldonio/seldon-core/operator/v2/pkg/cli"
+	"github.com/spf13/cobra"
+)
+
+func createUnload() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "unload resources",
+		Short: "unload resources",
+		Long:  `unload resources`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			flags := cmd.Flags()
+
+			schedulerHostIsSet := flags.Changed(flagSchedulerHost)
+			schedulerHost, err := flags.GetString(flagSchedulerHost)
+			if err != nil {
+				return err
+			}
+			authority, err := flags.GetString(flagAuthority)
+			if err != nil {
+				return err
+			}
+			showRequest, err := flags.GetBool(flagShowRequest)
+			if err != nil {
+				return err
+			}
+			showResponse, err := flags.GetBool(flagShowResponse)
+			if err != nil {
+				return err
+			}
+			filename, err := flags.GetString(flagFile)
+			if err != nil {
+				return err
+			}
+			var dataFile []byte
+			if filename != "" {
+				dataFile = loadFile(filename)
+			}
+
+			schedulerClient, err := cli.NewSchedulerClient(schedulerHost, schedulerHostIsSet, authority)
+			if err != nil {
+				return err
+			}
+
+			err = schedulerClient.Unload(dataFile, showRequest, showResponse)
```

Comment:
question: why we are calling `Unload` here?

Reply:
not sure I follow - this is the unload method